src/components/navigation-header/components/give-feedback-button/index.tsx

```diff
@@ -5,8 +5,8 @@ import ButtonLink from 'components/button-link'
 import s from './give-feedback-button.module.css'
 
 export const FORM_URL = 'https://forms.gle/fnHLuNahLEhjuKvE6'
-const LINK_TEXT = 'Give beta feedback'
-const LINK_ARIA_LABEL = 'Give beta feedback (opens in new tab)'
+const LINK_TEXT = 'Give feedback'
+const LINK_ARIA_LABEL = 'Give feedback (opens in new tab)'
```

Comment:
Would we then need to add a separate element with an `id` we'd reference in `aria-describedby`? 🤔 i guess i'm wondering the value here of `"opens in new tab"` (given that it might break voice control experiences, as you mentioned). Would we be okay with either leaving it out for now?

Reply:
Yep I'm fine with either of the two options I listed! (aside from the typo LOL)

Reply:
This is very close! Because the new element is _inside_ the `<a>` though, it's still included in the accessible name calculation. If we place the new element as a sibling to the `<a>`, give it an `id`, then pass that `id` to `<a>`'s `aria-describedby`, that should work!

Reply:
For these cases, I think our intent might be to include the `(opens in new tab)` in the accessible name calculation? That's my understanding of the guidance in [the aforementioned link](https://www.w3.org/TR/WCAG20-TECHS/G201.html).

Do you think it's necessary we use `aria-describedby` and a separate element in this case? If so, maybe the easiest way forward here would be to remove this work from this PR, and handle it separately.

Reply:
Update: have reverted changes here as I think they're adjacent to the contents of this PR rather than directly. Thinking it makes sense to pick up the a11y improvements we might need to make to this component in a separate PR 👍

Reply:
We can continue chatting about this in Slack! I'll make a little video recording demonstrating what I mean about voice control and the accessible name vs. the visual label

Reply:
Sweet, sounds good 💖 i am realizing now that what you'd meant is that we might need to intentionally exclude `(opens in new tab)` from the accessible label for voice control purposes. I think this could merit a broader effort to try to fix links like this throughout the site 👍 💯